### DISCUSSION TITLE
Make .csv file for ADO import part of repository

### DISCUSSION THREAD

**Discussion by FH-Inway** [microsoft/dynamics365patternspractices#129] [Ideas]
Many thanks for the guidance initiative!

I tried following the guidance to import the processes to Azure DevOps given here: https://learn.microsoft.com/en-us/dynamics365/guidance/business-processes/about-import-catalog-devops

However, I encountered some issues with the process and also had to fix several data errors in the .csv file. It would be good to make the .csv file part of this repository to enable more targeted feedback or pull requests.

**Comment by rachel-profitt** [comment]
Thanks for the feedback. We originally included the file in the GitHub repository; however, GitHub lacks the reporting we need to track how many downloads the file has. Now that the file is available in the Microsoft Download Center, you can use the GitHub Issues tab to create errors or catalog requests, which our team will pick up and process. If you could let us know the data errors and which version the errors are with, we will work to get them corrected.
